shared/src/test/diff/fcp/Church_CT.mls

```diff
@@ -1086,39 +1086,12 @@ to_church_ty = to_ch
 to_church_ty = to_ch_simplif
 //│ ╔══[ERROR] Inferred recursive type: 'a
 //│   where
-//│     'a :> forall 'b 'c 'd 'e. 'e -> ('b -> 'd
-//│       where
-//│         'a <: 'e -> 'b -> 'c
-//│         'e <: 'c -> 'd)
-//│ ╙──
-//│ anything -> 'a -> ('b -> 'c
-//│   where
-//│     'd <: 'a -> 'b -> 'e
-//│     'a <: 'e -> 'c)
-//│ where
-//│   'd :> forall 'f 'g 'h 'i. 'f -> ('g -> 'i
-//│     where
-//│       'd <: 'f -> 'g -> 'h
-//│       'f <: 'h -> 'i)
-//│   <:  to_church_ty:
-//│ int -> ChurchInt
-//│ ╔══[ERROR] Cyclic-looking constraint while typing def definition; a type annotation may be required
-//│ ║  l.1086: 	to_church_ty = to_ch_simplif
-//│ ║          	^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-//│ ╙── Note: use flag `:ex` to see internal error info.
-//│             = [Function: to_ch_simplif]
-
-// * Redefinition in the context of distrib, just for the record (still infers but doesn't check)
```

Comment:
Why have tests been removed? Did you mess up the merge?

Did you not review your test diffs?